testing/Travis/testcases/prototype/shouldFail/InvariantViolations/testinfo.yaml

```diff
@@ -1,3 +1,3 @@
 testCmds:
   - command: ampersand proto
-    exitcode: 10
+    exitcode: 20
```

Comment:
What do you intend to do with changing the expected exitcode here? As you can see from the logs, this results in errors.

Reply:
I had been reading the exit status codes in gitbook on https://ampersandtarski.gitbook.io/documentation/the-command-line-tool#exit-status.
It clearly states that exit code 20 is meant for violated invariant errors. That's why.
Could you please do a reality check on these codes for me?
I have fixed this for now to exitcode 10

Reply:
Exit code 20 is only fired whenever a database cannot be generated due to violation of an invariant:

~~~.haskell
validateRulesSQL :: (HasLogFunc env) => FSpec ->  RIO env [Text]
validateRulesSQL fSpec = do
    case filter (not . isSignal fSpec . fst) (allViolations fSpec) of
       []    -> return()
       viols -> exitWith . ViolationsInDatabase . map stringify $ viols
    hSetBuffering stdout NoBuffering
~~~